python/paddle/distributed/auto_parallel/api.py

```diff
@@ -142,3 +141,33 @@ def shard_tensor(
         return shard_tensor_static(
             data, dist_attr.process_mesh, dist_attr.sharding_specs
         )
+
+
+def dtensor_from_fn(fn, dist_attr, *args, **kwargs):
+    """
+    Construct a Distributed Tensor from a function of arguments.
+
+    Args:
+        fn (callable): A callable function that takes arguments of Distributed Tensor and returns tensor.
+        dist_attr (paddle.distributed.DistAttr): Specify how tensors are distributed or sliced on ProcessMesh.
+        *args (tuple): A tuple of arguments to be passed to the ``fn`` function.
+        **kwargs (dict): A dict of arguments to be passed to the ``fn`` function.
+
+    Retruns:
+        Tensor: A Tensor constructed from ``fn`` with distributed attributes.
+
+    Examples:
+
+    .. code-block:: python
+
+        >>> import paddle
+        >>> import paddle.distribute as dist
+        >>> # Create a distributed attribute
+        >>> mesh = dist.ProcessMesh([0, 1], dim_names=["x"])
+        >>> dist_attr = dist.DistAttr(mesh=mesh, sharding_specs=[None])
+        >>> # Call the function dtensor_from_fn with dist_attr parameter
+        >>> d_tensor = dist.dtensor_from_fn(paddle.ones, dist_attr=dist_attr, shape=[1])
+        >>> print(d_tensor)
```

Comment:
- code-block部分及以下整体往右加4个缩进
- 同时看ci检查，好像没有 `paddle.distribute`，应该改成 `paddle.distributed` 吧？务必注意代码示例要能跑通

```suggestion
    Examples:

        .. code-block:: python

            >>> import paddle
            >>> import paddle.distributed as dist
            >>> # Create a distributed attribute
            >>> mesh = dist.ProcessMesh([0, 1], dim_names=["x"])
            >>> dist_attr = dist.DistAttr(mesh=mesh, sharding_specs=[None])
            >>> # Call the function dtensor_from_fn with dist_attr parameter
            >>> d_tensor = dist.dtensor_from_fn(paddle.ones, dist_attr=dist_attr, shape=[1])
            >>> print(d_tensor)
```

Reply:
done,thx